packages/client/hmi-client/src/views/TA2Playground.vue

```diff
@@ -536,9 +797,29 @@ export default defineComponent({
 		<p>A playground for testing TA2 API integrations.</p>
 		<button type="button" @click="addPlace">Add place</button>
 		<button type="button" @click="addTransition">Add transition</button>
+		<button type="button" @click="createSampleModels">Create Models</button>
 		&nbsp;
 		<button type="button" @click="LotkaVolterra">LotkaVolterra</button>
 		<button type="button" @click="simulate">Simulate</button>
+		<form>
+			<label for="loadModel">
+				<input v-model="loadModelID" type="text" id="loadModelID" placeholder="Model ID" />
```

Comment:
We should remove these `id="XYZ"` attributes now that we're accessing the input values through v-model directly.